atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -120,8 +130,13 @@ def download_sources(self, sources, insecure=False, download_dir=SRPMS_DOWNLOAD_
 
         req_session = get_retrying_requests_session()
         for source in sources:
-            download_url(source['url'], dest_dir, insecure=insecure,
-                         session=req_session, dest_filename=source.get('dest'))
+            subdir = os.path.join(dest_dir, source.get('subdir', ''))
+            checksums = source.get('checksums', '')
```

Comment:
should should be either source.get('checksums')  to return None
or source.get('checksums', {}) 

because download_url is setting it to empty dict only if is None

Reply:
fixed it.